### ISSUE TITLE
QtWebEngine warnings with Qt5 on Windows 10

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#304]
On Windows 10, I typically see warnings like this on startup:
```
[0918/053926:ERROR:cache_util_win.cc(20)] Unable to move the cache: 5
[0918/053926:ERROR:cache_util.cc(134)] Unable to move cache folder C:\Users\edreamleo\AppData\Local\python\QtWebEngine\Default\GPUCache to C:\Users\edreamleo\AppData\Local\python\QtWebEngine\Default\old_GPUCache_000
[0918/053926:ERROR:cache_creator.cc(129)] Unable to create cache
[0918/053926:ERROR:shader_disk_cache.cc(589)] Shader Cache Creation failed: -2
```

These happen even when the VR plugins are all disabled.

Restarting the computer removes these warnings.   Ctrl-clicking on a url in Leo brings back the warnings.

**Comment by edreamleo** [comment]
Restarting the computer removes these warnings.  They may have been a strings edge case after I installed PyQt5.  I'll move the Ubuntu warnings to a new issue and close this.

**Comment by edreamleo** [comment]
Drat.  The warnings are back.

**Comment by edreamleo** [comment]
I've just asked the Qt people for help [here](https://forum.qt.io/topic/71413/cache-warnings-on-windows-10-pyqt-5-7-after-clicking-url-s-in-a-qtextbrowser).

**Comment by edreamleo** [comment]
Strange.  I get these warnings with some .leo files, but not others.

One way to get rid of the warnings is to delete the contents of the folder
~\AppData\Local\python\QtWebEngine.

And now I can't the message even after clicking a url in Leo.

**Comment by edreamleo** [comment]
Good news. The warnings happen only if a version of Leo running PyQt5 is already open.  Apparently, PyQt5 is holding the Python cache open even when Leo is not active.

So the workaround is to open leoPy.leo with Python 2, and do testing with Python 3.  Not great, but much better than seeing the dashed warnings.

**Comment by edreamleo** [comment]
Rev 69c3201 is a better workaround.  It leaves leoQt.py unchanged.  The viewrendered.py and bigdash.py plugins now use QTextBrowser widgets instead of QWebView widgets.  Apparently, instantiating a QWebView holds the cache open.

I am going to close this, but mark it Can'tFix and Waiting, to show that the workaround can be removed only after Qt5 issues a fix.

**Comment by edreamleo** [comment]
Recent revs instantiate the QWebView only when true html rendering is needed.

This seems to reduce warnings.
